dashboard/app/controllers/api/v1/sections_students_controller.rb

```diff
@@ -0,0 +1,13 @@
+class Api::V1::SectionsStudentsController < Api::V1::JsonApiController
+  load_and_authorize_resource :section
```

Comment:
For our new action, this is an implicit check that the current user `can :index, Section` and then sets instance variable `@sections` to the set of sections the user is allowed to `:index` - which is constrained to sections the user owns, [here in ability.rb](https://github.com/code-dot-org/code-dot-org/blob/staging/dashboard/app/models/ability.rb#L67) (note that `:manage` ability implies index, show, edit, destroy...).